src/components/LogsList/LogsList.js

```diff
@@ -31,6 +31,8 @@ export default class LogsList extends React.Component {
         interactive={false}
         maxHeight={800}
         onNeedMoreData={onNeedMoreLogs}
+        pageAmount={100}
```

Comment:
Same thing as in ui-agreements, can we move this to a constant we import from somewhere?